src/app/services/users-role-helper/users-role-helper.coffee

```diff
@@ -0,0 +1,29 @@
+@App.service 'UsersRole', (USER_ROLES, MnoeUsers, toastr, $translate) ->
+  _self = @
+
+  @availableRoles = _.map(USER_ROLES, 'value')
+
+  @keyFromRole = (role) ->
+    _.find(USER_ROLES, 'value', role).label
+
+  @updateUserRole = (organization, user, oldRole) ->
+    user.isUpdatingRole = true
+    # For mnoe-organizations-local-list we get the role from the user, whereas in mnoe-users-local-list, from the organization.
+    user.role = organization.role if organization.role
+    # Automatic binding not working in services.
+    userRole = this
```

Comment:
I had a weird binding error, where the fat arrow didn't automatically bind. Could not figure it out.